ad.md

```diff
@@ -1,6 +1,15 @@
-# Правила размещения рекламы
+# Правила размещения рекламы | Rules for advertising
 
-1) согласовать размещение рекламы с администраторами
-2) добавить ссылку на чат в инфопартнеры мероприятия на оф.сайте
+## Правила размещения рекламы
 
-Tip: Если Вы делаете рекламу в разных чатах, то для каждого чата необходимо выполнить оба пункта отдельно. Так же стоит учитывать, что согласие на рекламу дали только те чаты, где Вы согласовали рассылку и пин. Чаты не давшие согласие на рекламу, просьба не указывать и никак не упоминать.
+1. Согласовать размещение рекламы с администраторами
+2. Добавить ссылку на чат в инфопартнеры мероприятия на оф.сайте
+
+_Tip_: Если Вы делаете рекламу в разных чатах, то для каждого чата необходимо выполнить оба пункта отдельно. Так же стоит учитывать, что согласие на рекламу дали только те чаты, где Вы согласовали рассылку. Чаты не давшие согласие на рекламу, просьба не указывать и никак не упоминать.
+
+## Rules for advertising
+
+1. Coordinate the placement of advertising with administrators
+2. Add a link to the chat in the event’s info partners on the website
+
+_Tip_: If you are advertising in different chats, then for each chat you must complete both points separately. It is also worth considering that consent to advertising was given only by those chats where you agreed to the newsletter. Chats that do not consent to advertise, please do not indicate and do not mention.
```

Comment:
- you must complete both requirements for each chat.
- Please do not indicate and do not mention chats that have not consented to have advertisement placed